### DISCUSSION TITLE
optuna-dashboard not working (optuna.exceptions.StorageInternalError), help?

### DISCUSSION THREAD

**Discussion by annikanni** [optuna/optuna#4599] [Q&A]
Hi, 
I had run about 5 test studies and optuna-dashboard was working fine for us, but after running a longer XGBoost training (3-4 days) it has stopped working and gives the following error when trying to check the history: 

optuna.exceptions.StorageInternalError: An exception is raised during the commit. This typically happens due to invalid data in the commit, e.g. exceeding max length

The optuna-xgboost training finished without hickups, but now the storage is empty (every previous studies wiped out too). Do you know what is the problem and how to get optuna-dashboard working again? 

(optuna 3.1.0 but updated to 3.1.1 did not solve it, optuna-dashboard 0.9.0)

Thanks, 
Anni

**Comment by nzw0301** [comment]
Optuna-dashboard has its repository https://github.com/optuna/optuna-dashboard. You might get reply faster by asking the question there.

**Reply by annikanni** [reply]
Thanks! I'll try there too.

**Comment by annikanni** [comment]
But this is an optuna problem not saving the results into the sqlite file. So should it not be here?

**Comment by annikanni** [comment]
Oh sorry the directory where the sqlite file was saved changed and I did not notice it.
